addons/friendly_tracker/initSettings.sqf

```diff
@@ -0,0 +1,67 @@
+    false, // Extra properties of the setting depending of _settingType. <ANY>
+    false // 1: all clients share the same setting, 2: setting can't be overwritten (optional, default: 0) <NUMBER>
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(showUnconc), // Unique setting name. Matches resulting variable name <STRING>
+    "CHECKBOX", // Type of setting. Can be "CHECKBOX", "EDITBOX", "LIST", "SLIDER" or "COLOR" <STRING>
+    [LSTRING(ShowUnconc), LSTRING(ShowUnconc_Description)], // Display name or display name + tooltip (optional, default: same as setting name) <STRING, ARRAY>
+    LSTRING(DisplayName), // Category for the settings menu + optional sub-category <STRING, ARRAY>
+    false, // Extra properties of the setting depending of _settingType. <ANY>
+    true // 1: all clients share the same setting, 2: setting can't be overwritten (optional, default: 0) <NUMBER>
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(showVehicle), // Unique setting name. Matches resulting variable name <STRING>
+    "CHECKBOX", // Type of setting. Can be "CHECKBOX", "EDITBOX", "LIST", "SLIDER" or "COLOR" <STRING>
+    [LSTRING(showVehicle), LSTRING(showVehicle_Description)], // Display name or display name + tooltip (optional, default: same as setting name) <STRING, ARRAY>
+    LSTRING(DisplayName), // Category for the settings menu + optional sub-category <STRING, ARRAY>
+    false, // Extra properties of the setting depending of _settingType. <ANY>
+    true // 1: all clients share the same setting, 2: setting can't be overwritten (optional, default: 0) <NUMBER>
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(refreshRate), // Unique setting name. Matches resulting variable name <STRING>
+    "SLIDER", // Type of setting. Can be "CHECKBOX", "EDITBOX", "LIST", "SLIDER" or "COLOR" <STRING>
+    [LSTRING(RefreshRate), LSTRING(RefreshRate_Description)], // Display name or display name + tooltip (optional, default: same as setting name) <STRING, ARRAY>
+    LSTRING(DisplayName), // Category for the settings menu + optional sub-category <STRING, ARRAY>
+    [1, 600, 5, 0], // Extra properties of the setting depending of _settingType. <ANY>
+    true // 1: all clients share the same setting, 2: setting can't be overwritten (optional, default: 0) <NUMBER>
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(GPS), // Unique setting name. Matches resulting variable name <STRING>
+    "CHECKBOX", // Type of setting. Can be "CHECKBOX", "EDITBOX", "LIST", "SLIDER" or "COLOR" <STRING>
+    [LSTRING(GPS), LSTRING(GPS_Descrption)], // Display name or display name + tooltip (optional, default: same as setting name) <STRING, ARRAY>
+    LSTRING(DisplayName), // Category for the settings menu + optional sub-category <STRING, ARRAY>
+    false, // Extra properties of the setting depending of _settingType. <ANY>
+    true // 1: all clients share the same setting, 2: setting can't be overwritten (optional, default: 0) <NUMBER>
+] call CBA_fnc_addSetting;
```

Comment:
Can we get rid of these comments? This addon won't be edited by "dummies" like our mission template so I don't think they are necessary.

Reply:
Yes, I already removed them.